src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployerTests.java

```diff
@@ -1252,38 +1252,38 @@ public void testPodAntiAffinityPropertyOverrideGlobal() {
         assertThat(podAntiAffinityTest.getPreferredDuringSchedulingIgnoredDuringExecution().size()).as("PreferredDuringSchedulingIgnoredDuringExecution should have one element").isEqualTo(1);
     }
 
-	@Nested
-	@DisplayName("creates pod spec with pod security context")
+    @Nested
+    @DisplayName("creates pod spec with pod security context")
     class CreatePodSpecWithPodSecurityContext {
 
         @Test
         @DisplayName("created from deployment property with all fields")
         void createdFromDeploymentPropertyWithAllFields() {
             KubernetesDeployerProperties globalDeployerProps = new KubernetesDeployerProperties();
             Map<String, String> deploymentProps = new HashMap<>();
-			deploymentProps.put("spring.cloud.deployer.kubernetes.podSecurityContext", "{" +
```

Comment:
The spacing got janky across the cherry-pick. Going to leave it like this for now.